src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -126,13 +126,14 @@ Telemetry
 --------------
 The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community.
 You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell.
-You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.
-
-Configuring...
+You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.</value>
```

Comment:
@nguerrera I guess my question was more of the following: "If we are going to take this for 2.0, do we need to do anything in addition to the above (e.g. have the resources already been localized and so we'll have to split the localized data by hand as well?)